extension/intents/routines/routineExecutor.js

```diff
@@ -3,19 +3,22 @@
 import * as intentRunner from "../../background/intentRunner.js";
 
 export let pausedRoutineExecutor = null;
+export let currentRoutineExecutor = null;
```

Comment:
Huh, I didn't realize you could export a non-constant value. And from what I'm reading maybe it live updates when imported? Though that's a bit surprising to me that it works.

I'd usually use a getter function, but I suppose if this works...